modules/nw-metallb-configure-secondary-interface.adoc

```diff
@@ -0,0 +1,49 @@
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configure-secondary-interface_{context}"]
+= Configuring MetalLb with secondary networks
+
+From {product-title} 4.14 the default behavior is to not allow forwarding of IP packets between network interfaces. Therefore when MetalLb is configured on a secondary interfaces which is common practice you need to add a machine configuration to enable IP forwarding for only the required interfaces. 
+[NOTE]
+====
+{product-title} clusters upgraded from 4.13 are not affected as a global parameter is set during upgrade to enable global IP forwarding.
```

Comment:
```suggestion
{product-title} clusters upgraded from 4.13 are not affected because a global parameter is set during upgrade to enable global IP forwarding.
```